app/controllers/artists_controller.rb

```diff
@@ -153,42 +153,26 @@ def show
   end
 
   def update
-    if request.xhr?
```

Comment:
handle_xhr_request was only handling `os_status` updates which are now handled by their own endpoint so we don't really need this anymore.